snuba/query/mql/mql_context.py

```diff
@@ -0,0 +1,239 @@
+"""
+This file contains replicated classes from the snuba-sdk.
+"""
+from __future__ import annotations
+
+from dataclasses import dataclass, fields
+from datetime import datetime
+from typing import Any, Mapping, Optional, Sequence
+
+from snuba.query import OrderByDirection
+from snuba.query.mql.exceptions import InvalidExpressionError, InvalidMQLContextError
+
+
+class MQLContextValidator:
```

Comment:
I don't know if we need this class duplicated. There is probably a way to simplify this.